src/services/game-logic/Board.ts

```diff
@@ -11,7 +11,7 @@ import { Queen } from './pieces/Queen';
 class Board {
     public static BOARD_SIZE = 8;
 
-    private state: Array<Array<Piece | null>>;
+    public state: Array<Array<Piece | null>>;
```

Comment:
to jest chujowe, myślę co z tym zrobić

Reply:
ale widzę ten sam ruch w innym prze :P

Reply:
u mnie jest na public, bo sie bardzo przyadło :P